docs/configure.md

```diff
@@ -498,3 +498,7 @@ Sourcegraph's Kubernetes deployment [requires an Enterprise license key](https:/
 1. Create an account on or sign in to sourcegraph.com, and go to https://sourcegraph.com/subscriptions/new to obtain a license key.
 
 1. Once you have a license key, add it to your configuration by setting it [in the management console](https://docs.sourcegraph.com/admin/management_console).
+
+## Non-default namespace
+
```

Comment:
Doesn't grafana also need the same instructions?

Reply:
done